features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx

```diff
@@ -0,0 +1,303 @@
+/**
+ * Copyright (c)  WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
i think this is an old license.

Reply:
This is what we use now rather than updating the date I guess

Reply:
Fixed with ca7482b6a0e2231d3ac9a7e3cc01f30fcdfaa5fe